.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "e2e_accept/appLib/laTrappe"]
+	path = e2e_accept/appLib/laTrappe
+	url = git@github.com:kulicuu/laTrappe.git
```

Comment:
If a submodule is used I think it should be published under the socket stream org.

I don't see the need for a submodule rather than a subfolder